source/js/storage.js

```diff
@@ -0,0 +1,20 @@
+class NoOpStorage {
```

Comment:
"noop" is a common enough term that the "Op" part is rarely ever capitalized
```suggestion
class NoopStorage {
```